test-harness/build.gradle

```diff
@@ -15,6 +15,8 @@ dependencies {
     testImplementation project(':conductor-client')
     testImplementation project(':conductor-grpc-client')
 
+    testImplementation "org.springframework.retry:spring-retry"
```

Comment:
Don't we need also need this in `server/build.gradle` as `implementation`?

Reply:
I was able to bring up the server locally using `gradlew bootRun`. What am I overlooking?

Reply:
Added to server's build.gradle